activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -108,6 +113,8 @@ def self.build_mem_cache(*addresses) # :nodoc:
       #
       # If no addresses are provided, but <tt>ENV['MEMCACHE_SERVERS']</tt> is defined, it will be used instead. Otherwise,
       # MemCacheStore will connect to localhost:11211 (the default memcached port).
+      # Passing a Dalli::Client instance instead of addresses, is deprecated
+      # and will be removed.
```

Comment:
```suggestion
      ```suggestion

      # Passing a +Dalli::Client+ instance is deprecated and will be removed. Please pass an address instead.
```
```